src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/DiagramsEditor.kt

```diff
@@ -1,30 +1,48 @@
 package de.docs_as_co.intellij.plugin.drawio.editor
 
-import com.intellij.openapi.application.Application
 import com.intellij.openapi.application.ApplicationManager
-import com.intellij.openapi.command.WriteCommandAction
-import com.intellij.openapi.fileEditor.*
-import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.editor.colors.EditorColorsListener
+import com.intellij.openapi.editor.colors.EditorColorsManager
+import com.intellij.openapi.editor.colors.EditorColorsScheme
+import com.intellij.openapi.fileEditor.FileEditor
+import com.intellij.openapi.fileEditor.FileEditorLocation
+import com.intellij.openapi.fileEditor.FileEditorState
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.util.Key
 import com.intellij.openapi.vfs.VirtualFile
-import com.intellij.testFramework.runNonUndoableWriteAction
+import com.intellij.util.ui.UIUtil
 import com.jetbrains.rd.util.lifetime.LifetimeDefinition
 import java.beans.PropertyChangeListener
-import java.util.*
 import javax.swing.JComponent
+import javax.swing.UIManager
 
 
-class DiagramsEditor(private val project: Project, private val file: VirtualFile) : FileEditor {
+class DiagramsEditor(private val project: Project, private val file: VirtualFile) : FileEditor, EditorColorsListener {
     private val lifetimeDef = LifetimeDefinition()
     private val lifetime = lifetimeDef.lifetime
+    private var uiTheme = "dark" // "dark" or "kennedy"
```

Comment:
Can you make this an observable property? See how the `xml` field in the drawio instance is implemented!

Reply:
I though about an enumeration but at the moment, I just want to get it to work...